Changes for BOLD Report 10_28_25

jclina opened the pull request.

jclina committed:
Changes for BOLD Report 10_28_25

bhelsel committed:
Add new path to BRIDGE21.qmd after adding missing cognition variable

bhelsel committed:
Simplify APOE logic to reduce code down to one chunk

bhelsel committed:
Add missing dsmse variable to simplify code since kuadrc and bold are identical

bhelsel committed:
Add after_memory_section to simplify logic of where to include after-memory.tex

bhelsel committed:
Add an additional tryFormat to as.Date and move sex variable to intake form

bhelsel committed:
Add bold sub-reports to .subReports

bhelsel commented:
This is heading in the right direction! I made a few updates to the demographics, cognition, and apoe sections. I noticed that you reproduced the DS MSE table because it appeared that there was one less variable for BOLD. However, I saw that variable in the REDCap database and added it to the REDCap report. Also, I tried to simplify the logic and only used a new code chunk if necessary. You can also run conditional logic within the same code chunk if it makes sense (e.g., check out the updated APOE file). I still need to look at all the lifestyle files but feel free to make changes to them based on these suggestions if necessary.

jclina committed:
Added Vineland 10_29_25

 Attempting to add the Vineland for BOLD as practice for adding new material into the report.

jclina committed:
Attempt to simplify lifestyle logic down to one chunk

jclina committed:
Attempt to simplify bmi and dxa qmd files into one chunk

jclina committed:
Attempt to add VeggieMeter Section

jclina committed:
Attempt to add lipid and glucose table for BOLD

bhelsel committed:
Update .Rbuildignore

bhelsel committed:
Change .txt file extension and move table header

bhelsel committed:
Add calculate_bmi and format_date functions to make Quarto more flexible

bhelsel committed:
Slight improvement to plotting functions to align with Quarto changes

bhelsel committed:
Update NAMESPACE

bhelsel committed:
Remove the need for reports to be added the reports argument and instead allow them to be unquoted arguments given to ...

bhelsel committed:
Rename lipidglucose to metabolism to make it more flexible for future reports that desire to add variables

bhelsel committed:
Update blood.qmd

bhelsel committed:
Update Quarto with new format_date function

bhelsel committed:
Update activity.qmd

bhelsel committed:
Update Quarto with new calculate_bmi function

bhelsel committed:
Update BRIDGE21.qmd

jclina committed:
Updated NEWS.md and DESCRIPTION file.
